setup.py

```diff
@@ -10,7 +10,7 @@
         "hypothesis>=4.43.0,<5.0.0",
         "pytest>=6.2.5,<7",
         "pytest-xdist",
-        "tox==3.14.6",
+        "tox>=3.14.6,<4",
```

Comment:
Needed to open up the tox version to let docs build with 3.7

Reply:
Should/can we just remove the upper bound generally here?

Reply:
Upper bounds gone!